crates/holochain/src/core/queue_consumer/publish_dht_ops_consumer.rs

```diff
@@ -32,11 +34,23 @@ pub fn spawn_publish_dht_ops_consumer(
             let env = env.clone();
             let agent = cell_id.agent_pubkey().clone();
             let network = network.clone();
+            let min_publish_interval = conductor
+                .get_config()
+                .conductor_tuning_params()
+                .min_publish_interval
+                .unwrap_or(MIN_PUBLISH_INTERVAL);
```

Comment:
Recommend making a `min_publish_interval` function on the `ConductorTuningParams` that documents and provides the default. Like remove it from the inner workflow completely and have this be an input. 

This pattern is already in place for the first tuning param and hopefully makes it easier for people trying to work out how to tune the conductor to work out what the defaults are.

Reply:
I've refactored as you suggested.